internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -521,6 +521,60 @@ paths:
       security:
       - Bearer: []
       summary: Creates a Kafka request
+  /api/kafkas_mgmt/v1/supported_instance_types:
+    get:
+      operationId: getSupportedInstanceTypes
+      parameters:
```

Comment:
do those parameters belong to the endpoint? In ADR-81 cloud provider(e.g. `aws`) and region (e.g. `us-east-1`) path parameters are required. So we'd have to cross check the response with the supported cloud providers array.

Reply:
The way I can see it working is: you use [cloud provider and region](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/managed-services/cicd/saas/saas-kas-fleet-manager.yaml#L439) to get a list of supported instance types. Then you return the matching instance types from [supported instance types](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/managed-services/cicd/saas/saas-kas-fleet-manager.yaml#L126).

Reply:
Thanks for clarification @pb82